contracts/icns-resolver/src/crypto.rs

```diff
@@ -76,6 +76,9 @@ pub fn create_adr36_message(
     contract_address: String,
     signature_salt: u128,
 ) -> String {
+    // NL: Consuder using raw strings and interpolation here/
+    // Alternatively, serde_json_wasm::to_string() and a struct.
+    // This way of doing it should be safe though. As this is only used for validation
```

Comment:
what's the advantage of using raw strings?

Reply:
just easier to read / harder to introduce typos when modifying it. This function becomes:

`format!(r#"{"account_number":"0","chain_id":"","fee":{"amount":[],"gas":"0"},"memo":"","msgs":[{"type":"sign/MsgSignData","value":{"data":"{data}","signer":"{signer_bech32_address}"}}],"sequence":"0"}"#)`